ansible_collections/arista/avd/examples/campus-fabric/documentation/devices/LEAF1B.md

```diff
@@ -76,16 +76,16 @@ interface Vlan10
    ip address 10.10.10.7/24
 ```
 
-### Name Servers
+### IP Name Servers
```

Comment:
IMO the headings do not have to match the key 1:1. Example under access-lists the key is ip_access_lists.

Reply:
so they do not have to but what decision should we take here? (if this is only to avoid changes to doc I don't think that's a good argument)